pkg/command/root.go

```diff
@@ -471,6 +472,18 @@ var opsCmd = &cobra.Command{
 	},
 }
 
+var tpeCmd = &cobra.Command{
```

Comment:
How about putting a comment explaining that this command is not a target but that plugins can provide sub-commands by using 
```
CommandMap: []plugin.CommandMapEntry{
   {DestinationCommandPath: "platform-engineering myplugin"},
},
```

Reply:
I pointed to the doc section that mentions this use case instead, hope that's is okay as well.